test/functional/wallet_transactiontime_rescan.py

```
@@ -14,13 +14,20 @@
+    get_generate_key,
+)
 
 
 class TransactionTimeRescanTest(BitcoinTestFramework):
     def set_test_params(self):
         self.disable_mocktime = True
         self.setup_clean_chain = False
         self.num_nodes = 3
+        self.extra_args = [["-keypool=400"],
+                           ["-keypool=400"],
+                           []
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)